### ISSUE TITLE
Run tests with Spring profile

### ISSUE THREAD

**Issue by iuadastra** [microsoft/vscode-spring-boot-dashboard#337]
I can start Spring Boot app using "Run with Profile..." which is great. However it's not possible to run tests with some profiles. Would be great to have such feature.

**Comment by brunovieira97** [comment]
From what I know, tests are handled by the [vscode-java-test](https://github.com/microsoft/vscode-java-test) project.

That project seems to be completely agnostic to Spring or other frameworks, and so I believe handling Spring profiles for test execution would prove to be a challenge. Despite that, it's worth to open an issue there and see which ideas come from a discussion on this matter.

**Comment by testforstephen** [comment]
Yes, the run tests capability is provided by test runner for Java extension. It's better to open an issue on the tester extension to discuss how to support this.

**Comment by kaaapi** [comment]
While this is indeed a limitation with the vscode-java-test as precisely mentioned, a workaround (although not a very GUI one) without needing to configure a dedicated maven profile, one can add the following environment config to vscode-java-test as follows which does enable preferred spring profile for tests and easily debug as usual via test-runner:

**File:** `.vscode/settings.json`

```json
"java.test.config": {
    "env": {
        "SPRING_PROFILES_ACTIVE": "<profile-name>"
    }
}
```
